components/automate-cli/pkg/verifyserver/handlers/checks/checks.go

```diff
@@ -0,0 +1,15 @@
+package checks
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+)
+
+type Checks struct {
+	Logger logger.ILogger
+}
+
+type IChecks interface{}
```

Comment:
Not very clear what is the need of the interface here. Since we are creating handler for all the endpoints which are unique by the request body

Reply:
This is to support backwards compatibility. Later on if ve have v1/v2 versioning of apis,, interfaces can be helpful to implement the apis

Reply:
I don't think for API versioning we need to have Interfaces, @vivekshankar1  connect with me to discuss this more.

Reply:
Removed the implementation with interfaces